xds/src/main/java/io/grpc/xds/internal/sds/XdsServerBuilder.java

```diff
@@ -171,9 +173,27 @@ void panic(final Throwable t) {
               panicMode = true;
             }
           });
-    delegate.protocolNegotiator(
+    InternalProtocolNegotiator.ProtocolNegotiator serverProtocolNegotiator =
         SdsProtocolNegotiators.serverProtocolNegotiator(
-            this.downstreamTlsContext, port, syncContext));
-    return delegate.build();
+            this.downstreamTlsContext, port, syncContext);
+    return createXdsServer(serverProtocolNegotiator);
```

Comment:
it is weird that this build() doesn't return XdsServer even though it is XdsServerBuilder. Although it is questionable that we need that XdsServer class or not.

Reply:
As mentioned above, we need to keep the `XdsClientWrapperForServerSds` instance thru the life of the server and shut down when the server shuts down. It can always return XdsServer even if not needed just to avoid the "weirdness"?